index.html

```diff
@@ -720,9 +738,9 @@ <h2>Profiling Mechanism</h2>
 
   <!-- HTTP Profiles -->
   <section id="http-profiles">
-    <h2>HTTP Profiles</h2>
+    <h2>HTTP Profile</h2>
     <p>
-      The HTTP Profile specificiation defines several profiles that can be used for dedicated use cases.
+      The HTTP Profile specification defines several profiles that can be used for dedicated use cases.
```

Comment:
There is no HTTP Profile specification, only the WoT Profile Specification afaik